src/index.js

```diff
@@ -134,6 +136,18 @@ const Spruce = {
                     return comparison[part]
                 }, stores)
             })
+    },
+
+    persistUsing(driver) {
+        if (driver.getItem === undefined) {
+            throw new Error('[Spruce] The persistence driver must have a `getItem(key)` method.')
+        }
+
+        if (driver.setItem === undefined) {
```

Comment:
how about using `typeof driver.setItem === 'function'`?

Reply:
Good shout. Don't know why I didn't do that first 🤦‍♂️